app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -158,6 +173,13 @@
                                   track_label: item[:href],
                                   track_dimension: item[:label],
                                   track_dimension_index: "29",
+                                  ga4_link: {
+                                      "event_name": "navigation",
+                                      "type": "header menu bar",
+                                      "index": '1.' + (column_index + 1).to_s + '.' + (index + 1).to_s,
```

Comment:
Instead of calling `.to_s` twice, I think you could do

```Ruby
"index": "1.#{column_index + 1}.#{index + 1}",
```

(similar for line 320)

Reply:
Thanks :+1: